chia/simulator/setup_services.py

```diff
@@ -119,17 +119,18 @@ async def setup_full_node(
     *,
     reuse_db: bool = False,
 ) -> AsyncGenerator[Union[Service[FullNode, FullNodeAPI], Service[FullNode, FullNodeSimulator]], None]:
-    db_path = local_bt.root_path / f"{db_name}"
-    if not reuse_db and db_path.exists():
-        # TODO: remove (maybe) when fixed https://github.com/python/cpython/issues/97641
-        gc.collect()
-        db_path.unlink()
-
-        if db_version > 1:
-            with sqlite3.connect(db_path) as connection:
-                connection.execute("CREATE TABLE database_version(version int)")
-                connection.execute("INSERT INTO database_version VALUES (?)", (db_version,))
-                connection.commit()
+    if reuse_db:
```

Comment:
does `reuse_db` mean that we want to be able to close the connection, and then open up a new connection?
if it just means we want multiple connections to it, you can do that with an in-memory DB as well, you just need to add `&cache=shared` to the URI